va_explorer/templatetags/va_explorer_tags.py

```diff
@@ -11,6 +12,27 @@
 
 register = template.Library()
 
+@register.filter
+def replace(value):
+    value = value.strip()
```

Comment:
I think you can remove the parenthesis with these conditionals: https://google.github.io/styleguide/pyguide.html#33-parentheses

Reply:
do you mean remove the parentheses such that its `value.strip` and not `value.strip()` ?

Reply:
No, sorry for being unclear. I mean `if("dk" == value or "DK" == value)` --> `if "dk" == value or "DK" == value`

This is purely stylistic.